cishouseholds/merge.py

```diff
@@ -30,7 +34,31 @@ def flag_identical_rows_after_first(df: DataFrame, exclusion_columns: Union[List
     return df.drop("ROW_NUMBER")
 
 
-def union_multiple_tables(tables: List[DataFrame]):
+def union_tables_hadoop(tables: List[DataFrame], reorder_columns: Optional[bool] = True):
```

Comment:
refactor to use new prepare for union function that recieves all tables then returns column order